src/utils/parseTable.js

```diff
@@ -297,22 +304,25 @@ export function htmlToGridSettings(element, rootDocument = document) {
 
           return settings;
         }, {});
+        let cellValue = '';
 
         if (cellStyle.whiteSpace === 'nowrap') {
-          dataArr[row][col] = innerHTML.replace(/[\r\n][\x20]{0,2}/gim, '\x20')
+          cellValue = innerHTML.replace(/[\r\n][\x20]{0,2}/gim, '\x20')
             .replace(/<br(\s*|\/)>/gim, '\r\n')
-            .replace(/(<([^>]+)>)/gi, '')
-            .replace(/&nbsp;/gi, '\x20');
+            .replace(regSpace, '\x20');
 
         } else if (generator && /excel/gi.test(generator.content)) {
-          dataArr[row][col] = innerHTML.replace(/[\r\n][\x20]{0,2}/g, '\x20')
+          cellValue = innerHTML.replace(/[\r\n][\x20]{0,2}/g, '\x20')
             .replace(/<br(\s*|\/)>[\r\n]?[\x20]{0,3}/gim, '\r\n')
-            .replace(/(<([^>]+)>)/gi, '')
-            .replace(/&nbsp;/gi, '\x20');
+            .replace(regSpace, '\x20');
+
         } else {
-          dataArr[row][col] = innerHTML.replace(/<br(\s*|\/)>[\r\n]?/gim, '\r\n').replace(/(<([^>]+)>)/gi, '').replace(/&nbsp;/gi, '\x20');
+          cellValue = innerHTML.replace(/<br(\s*|\/)>[\r\n]?/gim, '\r\n')
+            .replace(regSpace, '\x20');
         }
 
+        dataArr[row][col] = cellValue.replace(/(&lt;)|(&gt;)|(&amp;)/gi, match => ESCAPED_HTML_CHARS[match]);
```

Comment:
```suggestion
        const regEx = new RegExp(Object.keys(ESCAPED_HTML_CHARS).map(key => '(' + key + ')').join('|'), 'gi');
        dataArr[row][col] = cellValue.replace(regEx, match => ESCAPED_HTML_CHARS[match]);
```
Sorry, couldn't help myself. If one part is based on the `ESCAPED_HTML_CHARS` and the second is hardcoded this might introduce an uninteded error in the future if someone forget one or another.